conf/docker/dataverse-glassfish/entrypoint.sh

```diff
@@ -109,29 +113,26 @@ if [ "$1" = 'dataverse' ]; then
     fi
     
     GLASSFISH_INSTALL_DIR="/usr/local/glassfish4"
-    cd $GLASSFISH_INSTALL_DIR
-    cp /tmp/dvinstall.zip $GLASSFISH_INSTALL_DIR
-    unzip dvinstall.zip
-    cd dvinstall
+    cd /usr/local/glassfish4/dvinstall
     echo Copying the non-interactive file into place
     cp /tmp/default.config .
     echo Looking at first few lines of default.config
     head default.config
     # non-interactive install
     echo Running non-interactive install
     #./install -y -f > install.out 2> install.err
-    ./install -y -f
+    ./startup -y -f 
```

Comment:
We don't want to maintain `startup` as well as `install` do we?

Reply:
@pdurbin I understand that and I have yet to corporate the divided scripts (populate and startup) to /scripts/installer. Will start asap